src/sync.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Runs the given command in each repository, and commits
+ * files that were added or changed.
+ */
+
+'use strict';
+
+import * as util from 'util';
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+import * as cp from 'child_process';
+import * as meow from 'meow';
+import {getConfig} from './lib/config';
+import {GitHub} from './lib/github';
+import * as logger from './lib/logger';
+
+const mkdir = util.promisify(fs.mkdir);
+const readdir = util.promisify(fs.readdir);
+const stat = util.promisify(fs.stat);
+
+function exe(command: string, args: string[], options: cp.SpawnOptions = {}) {
```

Comment:
What does exe stand for?

Reply:
I understand that it's meant to mean 'execute' but I'd rather rename it to `spawn` to make it just a little bit more readable. I'm OK with both options though.

Reply:
switched to spawn